test/run_tests.sh

```diff
@@ -108,42 +108,6 @@ function run_xla_backend_mp {
   MASTER_ADDR=localhost MASTER_PORT=6000 run_test "$@"
 }
 
-function run_xrt {
-  if [ -x "$(command -v nvidia-smi)" ] && [ "$XLA_CUDA" != "0" ]; then
-    GPU_NUM_DEVICES=2 run_coverage "$@"
-  else
-    XRT_DEVICE_MAP="CPU:0;/job:localservice/replica:0/task:0/device:XLA_CPU:0" XRT_WORKERS="localservice:0;grpc://localhost:$(shuf -i 40701-40999 -n 1)" run_coverage "$@"
-  fi
-}
-
-function run_opbyop {
-  echo "Running in OpByOp mode: $@"
-  XLA_GET_TENSORS_OPBYOP=1 XLA_SYNC_TENSORS_OPBYOP=1 run_xrt "$@"
-}
-
-function run_async_scalar {
-  echo "Running in Async Scalar Upload mode: $@"
-  XLA_TRANSFER_SCALAR_ASYNC=1 run_xrt "$@"
-}
-
-function run_torchrun {
-  echo "Running tests spawned by torchrun"
-  if [ -x "$(command -v nvidia-smi)" ]; then
-    run_xrt "$@"
-  else
-    echo "the tests need atleast two XLA workers to validate"
-  fi
-}
```

Comment:
why are these removed?

Reply:
`run_async_scalar` is only used in `run_xrt_tests`,
`run_torchrun` is only used in `run_xrt_tests` too :
 https://github.com/pytorch/xla/blob/b3f4c7bc1e9f812069770c83e38e5bd2dc20aef6/test/run_tests.sh#LL138C10-L138C23 

so deleted `run_async_scalar` and `run_torchrun` when delete `run_xrt_tests`